frb/scripts/pz_dm.py

```diff
@@ -16,6 +16,8 @@ def parser(options=None):
                         help="Confidence limits for the z estimate [default is a 95 percent c.l., (2.5,97.5)]")
     parser.add_argument("--magdm_plot", default=False, action='store_true', 
                         help="Plot the host redshift range given DM on the magnitude vs redshift evolution")
+    parser.add_argument("--perfect_telescope", default=False, action='store_true', help="Use a perfect telescope model for the DM-z grid instead of CHIME")
```

Comment:
let's change this to be `--telescope`
with "Ideal" or something like that as the default
and `CHIME`, `DSA`, `ASKAP/ICS`, `ASKAP/CRACO` etc.
as options

Reply:
done